deploy/obo-removal/01-ClusterRole.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sre-operator-reinstall-sa
+  namespace: openshift-observability-operator
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
Some overhead in permissions here as I copy pasted it from another cronjob, happy to remove but won't hurt us.